.github/workflows/server-node.yml

```
@@ -17,14 +17,14 @@ jobs:
     strategy:
       matrix:
         # Node versions to run on.
-        version: [16, 18]
+        version: [18, 20]
```

```suggestion
        version: [18, 24]
```

I think we should be able to do this now.